src/Files.App/Actions/BaseUIAction.cs

```diff
@@ -11,6 +12,8 @@ internal abstract class BaseUIAction : ObservableObject, IAction
 
 		public abstract string Description { get; }
 
+		public virtual RichGlyph Glyph { get; } = RichGlyph.None;
+
```

Comment:
HotKey should be declared as well. Otherwise, hotkeys will not be enabled in the same way as icons.
I rather think IAction should be implemented in each concrete Action, not in BaseUIAction. This way, there is no need to declare the same default implementation in IAction and BaseUIAction.

Reply:
@ferrariofilippo

Reply:
> HotKey should be declared as well. Otherwise, hotkeys will not be enabled in the same way as icons. I rather think IAction should be implemented in each concrete Action, not in BaseUIAction. This way, there is no need to declare the same default implementation in IAction and BaseUIAction.

Yes it makes sense. I didn't think that we had to declare fiels in BaseUIAction too when I first created it, anyway I'll revert it now since it will make things clearer